docker/localsettings.py

```diff
@@ -251,3 +250,7 @@
     SHARED_TEMP_DIR_NAME = 'temp'
 
 BIGCOUCH = True
+
+LOCAL_APPS = (
+    'kombu.transport.django',  # required for celery
```

Comment:
I don't understand why this is needed, and I find it a bit odd that it's needed here and not in other places. Should it go in `settings.py`? If not, why not?

Reply:
I'm not sure - I think we may have a different configuration in non-local environments?  Aliza just PR-ed a change to the readme that suggests the same thing.